.travis.yml

```diff
@@ -0,0 +1,25 @@
+env:
+  - NODE_VERSION=0.10
+  - NODE_VERSION=0.12
+
+os:
+  - linux
+  - osx
+
+install:
+  - git clone https://github.com/creationix/nvm.git /tmp/.nvm
+  - source /tmp/.nvm/nvm.sh
+  - nvm install $NODE_VERSION
+  - nvm use $NODE_VERSION
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
+      sudo apt-get install build-essential git libgnome-keyring-dev fakeroot;
+      export DISPLAY=:99.0;
+      sh -e /etc/init.d/xvfb start;
+    fi
+
+script: script/cibuild
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
```

Comment:
Agreed :+1: 

Probably not a significant speedup (since Travis already uses a depth of `50`), but it's nice to be consistent with other repos :+1: